doctr/local.py

```diff
@@ -248,3 +248,20 @@ def check_repo_exists(deploy_repo, service='github', *, auth=None, headers=None)
     r.raise_for_status()
 
     return r.json().get('private', False)
+
+def in_git_root():
+    """
+    Check if current working directory is root dir of git repo.
+    Return ``False`` if it isn't.
+    """
+    repo = subprocess.run(['git', 'rev-parse', '--show-toplevel'],
```

Comment:
OTOH, maybe weird stuff will happen when checking out gh-pages if we aren't in the root (unless the directory we are in isn't tracked, or is in both branches).